elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -105,7 +117,23 @@ def attach_cve_flaws_cli(runtime, advisory_id, noop, default_advisory_type):
     if noop:
         return True
 
-    return advisory.commit()
+    advisory.commit()
+
+    # Associate builds with CVEs
```

Comment:
Could this be in a dedicated method, and put under test?

Reply:
Good idea. I haven't added any unit tests yet. This card is more complex than I estimated :(
Will add unit tests in the next push.